docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,135 @@
+---
+id: running-and-editing-timelines
+title: Running and Editing Timelines
+sidebar_label: Running and Editing Timelines
+---
+
+Running a timeline to timecode
+------------------
+
+The HTP (intensity) of fixtures in the timeline is controlled by the fader level of the handle. 
+
+1.  Ensure the **timecode link** button (bottom left of the timeline view) is set to **Linked**.
+
+2.  Push up the fader of the handle where the timeline is stored.
```

Comment:
This step is no longer required if you are using the "Activate in range" option which is enabled by default.